packages/flutter/lib/src/material/page.dart

```diff
@@ -48,13 +48,14 @@ class _CupertinoPageTransition extends AnimatedWidget {
 
```

Comment:
while we're here, can you rename _kTween to something that says what it is (e.g. _kHorizontalSlideTween) and make it a constant? (can it be a constant?)